packages/abstract-core/src/native/version_control.rs

```diff
@@ -101,6 +135,13 @@ pub enum QueryMsg {
         limit: Option<u8>,
         yanked: bool,
```

Comment:
This is a different change but this should be in the `ModuleFilter`

Reply:
@adairrr 
Here's the ambiguous thing.
So filter param is Option<ModuleFilter>. If this is None, then should we paginate the merged map of both yanked and normal modules?